src/core/scene/a-scene.js

```diff
@@ -296,8 +296,12 @@ var AScene = module.exports = registerElement('a-scene', {
 
         // setTimeout in case the camera is being set dynamically with a setAttribute.
         setTimeout(function checkForCamera () {
-          var sceneCameras = self.querySelectorAll('[camera]');
-          if (sceneCameras.length !== 0) { return; }
+          var camera = self.querySelector('[camera]');
```

Comment:
I know it's unlikely, but let's hope nobody else has a `camera` attribute on any other custom elements